Resolves #185 - Flesh out user scenarios Pt 1

mafriend opened the pull request:
## The Problem/Issue/Bug:
We currently do not have documentation around how DDEV UI operates and therefore cannot use the scenarios for testing/development purposes.

## How this PR Solves The Problem:
Bit by bit we are going to flesh out all the user scenarios for DDEV UI. This is the first pr of many as we fill out this feature file! 

@alkymst any and all feedback on how we can better validate that something has stopped/started/been created other than just looking at the UI would be super awesome! 

## Related Issue Link(s):
https://github.com/drud/ddev-ui/issues/185

dclear commented:
@alkymst : when you get a chance, can you review this one? ty

alkymst commented on feature-testing/test/features/resources/all-ui.feature:
```diff
@@ -0,0 +1,64 @@
+Given I have DDEV UI installed
+And Docker is running
+When I open up DDEV UI
+Then the command line Docker check passes
```
So it' not a command line check for docker. It uses the `ddev list` command which under the hood has checks for Docker. So it's more you will pass them initial application system checks and land inside the app when docker is installed and running, if not it will tell you that there was a problem connecting to Docker.

mafriend replied:
Ahh gotcha! This scenario and the other one you commented on in the same feature file havent been filled out yet (theyre pulled right from kory's doc) but ill pop these notes in there right now!

alkymst commented on feature-testing/test/features/resources/all-ui.feature:
```diff
@@ -0,0 +1,64 @@
+Given I have DDEV UI installed
+And Docker is not running
+When I open up DDEV UI
+Then the command line Docker check fails
```
So you would not pass the system check screen

mafriend replied:
updated the verbage here!

alkymst commented on feature-testing/test/features/resources/basic-ddev-commands.feature:
```diff
@@ -0,0 +1,28 @@
+Scenario: Can restart a DDEV project
+    And the user has a project running
+    When the user clicks the "Restart" button
+    ##TODO whats a good way to tell that the project has been restarted? 
```
you will see it stop and then restart visually in the app

alkymst commented:
@mafriend SO I added some comments and it looks like we have a conflict with merging ^ Let me know if you need me to take a look to resolve the conflict.

mafriend commented:
Hey @alkymst ! I've gone ahead and addressed your feedback as well as fixing the merge conflicts! I'm going to go ahead and merge this in since it was approved a few days ago, but if you have any feedback on the changes, please let me know!
